commercial/app/controllers/commercial/HostedContentController.scala

```diff
@@ -10,14 +10,28 @@ import views.html.hosted.{episode1Video, guardianHostedPage, teaserVideo}
 object HostedContentController extends Controller {
 
   def renderHostedPage(pageName: String) = Action { implicit request =>
-    val page = HostedPage(pageName)
+    val pageUrl = controllers.commercial.routes.HostedContentController.renderHostedPage(pageName).absoluteURL
```

Comment:
No it's not. It's only going to be needed when the `pageName` matches the `case matches` :smile: